dbt/adapters/redshift/relation.py

```diff
@@ -60,31 +59,28 @@ def relation_max_name_length(self):
         return MAX_CHARACTERS_IN_IDENTIFIER
 
     @classmethod
-    def from_runtime_config(cls, runtime_config: RuntimeConfigObject) -> RelationConfigBase:
-        model_node: ModelNode = runtime_config.model
-        relation_type: str = model_node.config.materialized
+    def from_config(cls, config: RelationConfig) -> RelationConfigBase:
```

Comment:
Should this be `from_relation_config`? It looks like we went with that in other places.

Reply:
We can resolve this in the future if needed. It's a naming convention concern, not a functionality concern.